components/game/game.js

```diff
@@ -103,6 +110,43 @@ const Game = () => {
     setComboStatus(true);
   }
 
+  /**
+   * @description Simulates user selecting combos.
+   */
+  const comboChoiceLoop = () => {
+    const randomInt = Math.floor(Math.random() * (6 - 1 + 1) + 1);
```

Comment:
as discussed, remove this `randomInt` logic and just show all the combo choice options with the ability to choose one. you'll need to add that to state. we can either comment this all out or fully remove it later after testing is done.